libraries/standard/http/src/http_client.c

```diff
@@ -7,6 +7,22 @@
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief An aggregator that represents the user-provided parameters to the
+ * #HTTPClient_ReadHeader API function, to be used as context parameter
+ * to the parsing callback used by the API function.
+ */
+typedef struct readHeaderContext
+{
+    const uint8_t * pHeaderName;
+    size_t headerNameLen;
+    const uint8_t ** pHeaderValueLoc;
+    size_t * pHeaderValueLen;
+    uint8_t headerFound;
+} readHeaderContext_t;
```

Comment:
This is not needed anymore?

Reply:
This needs to be removed if not needed.

Reply:
Removed. Was the artifact of the prior implementation of `ReadHeader`